### PR TITLE
Account form

### PR DESCRIPTION
**Pull request by tomve1tch**
-Created the account table in the database
-Created the account form. Linked form to database
-Displays most recent account information submission by current user on their account page

### CODE REVIEW

**Comment by tomve1tch** [comment]
Currently, there is redundancy on the Account page; with there being multiple times that the same information is being displayed (due to conflicts in the code). This will later be resolved on the styles branch according to how the front end team want to display the information stored in the db.

File: app/forms.py [hunk 1]
```diff
@@ -50,4 +50,14 @@ class JobForm(FlaskForm):
...
+    submit = SubmitField("Create")
+
+
+class ApplyForm(FlaskForm):
+    title_choices = [('Mr.', 'Mr.'), ('Ms.', 'Ms.'), ('Mrs.', 'Mrs.'), ('Miss.', 'Miss.'), ('Master.', 'Master.'), ('Madam.', 'Madam.'), ('Mx.', 'Mx.')]
+    title_apl = SelectField('Preferred Title', choices=title_choices, validators=[DataRequired()])
+    health_choices = [('Physically Able','Physically Able'), ('Not Physically Able','Not Physically Able'), ('I Would Rather Not Say', 'I Would Rather Not Say')]
+    health = SelectField('Health Status', choices=health_choices, validators=[DataRequired()])
+    earliest_start_date = DateField('Earliest Start Date', default=datetime.now(timezone.utc))
+    personal_bio = TextAreaField('Personal Bio', validators=[DataRequired()])
```
Comment (saundlj) [inline app/forms.py:62]: should we put a max or min string length for the bio? looks good though

File: app/models.py [hunk 2]
```diff
@@ -57,9 +59,16 @@ class Post(db.Model):
...
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False, default = 1) # lowercase becuase referencing column NOT class
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False, default = 1) # lowercase because referencing column NOT class
     #user_id logic needs to be fixed
 
     def __repr__(self):
         return f"Post('{self.title}', '{self.date_posted})"
-    
\ No newline at end of file
+
+class Account(db.Model):
```
Comment (saundlj) [inline app/models.py:68]: nice addition of account class. I think just the one relationship with user is a good call.

**Review by saundlj — APPROVED** [review]
Looks really good mate. No merge conflicts so happy to approve.

**Pull request merged**